.github/workflows/codeql-analysis.yml

```diff
@@ -29,10 +29,13 @@ jobs:
         with:
           fetch-depth: 0
 
-      # If this run was triggered by a pull request event, then checkout
-      # the head of the pull request instead of the merge commit.
-      - run: git checkout HEAD^2
-        if: ${{ github.event_name == 'pull_request' }}
+      # 20210407: 1 issue was detected with this workflow: git checkout HEAD^2 is no longer necessary.
+      # Please remove this step as Code Scanning recommends analyzing the merge commit for best results.
+
+      # # If this run was triggered by a pull request event, then checkout
+      # # the head of the pull request instead of the merge commit.
+      # - run: git checkout HEAD^2
+      #   if: ${{ github.event_name == 'pull_request' }}
```

Comment:
Can we delete this rather than commenting it out?

Reply:
Sure, np.